src/main/java/io/fabric8/elasticsearch/plugin/filter/FieldStatsResponseFilter.java

```diff
@@ -72,6 +73,17 @@ public void apply(final Task task, final String action, final ActionRequest requ
             @SuppressWarnings("unchecked")
             @Override
             public void onResponse(ActionResponse response) {
+                if(INDICES_FIELD_STATS_READ_ACTION.equals(action) && response instanceof FieldStatsResponse) {
+                    if(((FieldStatsResponse)response).getIndicesMergedFieldStats().size() == 0) {
+                        LOGGER.trace("Modifying the response to be {}", RestStatus.NOT_FOUND);
+                        Throwable err = new ElasticsearchException("The index is empty. This could mean data has not yet been collected.\n"
```

Comment:
"...This could mean data has not yet been collected..." Is this message correct?  From an openshift perspective, I would expect this index to ONLY exist if the collector has created it.

Reply:
You are right, basically we need a message to explain the user that he need to choose a larger time or something like that, do you have a proposal?

Reply:
Maybe: "There were no results found.  You may need to modify the time range by using the Time Picker to select a different interval."